ngraph/frontend/tensorflow/src/utils.hpp

```diff
@@ -138,117 +69,16 @@ static void MakePadding(const std::string& tf_padding_type,
-                << ")";
-            return errors::InvalidArgument(buf.str());
-        }
-    }
-    return Status::OK();
-}
-
-//
-// Helper for storing ops in ng_op_map.
-// For most of the cases, op would have one output so
-// std::vector ng_op_map[op_name] would contain one element.
-//
-// If storing more than one output_nodes, make sure it's in
-// the same order as tensorflow would do that.
-//
-// Parameters:
-//    Builder::OpMap& ng_op_map        - The TF-to-nGraph op map.
-//    std::string op_name              - Name of the op.
-//
-//    ov::Output<ov::Node> output_node - ov::Node to store
-//
-static void SaveNgOp(OpMap& ng_op_map, const std::string& op_name, ov::Output<ov::Node> output_node) {
-    // no need to try-catch, map[key] will create std::vector object
-    // if not exists
-    ng_op_map[op_name].push_back(output_node);
-}
-
-template <class TOpType, class... TArg>
-ov::Output<ov::Node> ConstructNgNode(const std::string& op_name, TArg&&... Args) {
-    auto ng_node = std::make_shared<TOpType>(std::forward<TArg>(Args)...);
-    SetTracingInfo(op_name, ng_node);
-    return ng_node;
-}
-
-static Status GetInputNode(const NodeContext op, size_t input_idx, ov::Output<ov::Node>& result) {
-    // Stub
-    result = op.get_ng_input(input_idx);
-    return Status::OK();
-}
-
-namespace detail {
-static Status GetInputNodes(const NodeContext&, size_t) {
-    return Status::OK();
-}
-
-template <typename... Arguments>
-static Status GetInputNodes(const NodeContext node,
-                            size_t index,
-                            ov::Output<ov::Node>& result,
-                            Arguments&... remaining) {
-    TF_RETURN_IF_ERROR(GetInputNode(node, index, result));
-    return GetInputNodes(node, index + 1, remaining...);
-}
-}  // namespace detail
-
-template <typename... Arguments>
-static Status GetInputNodes(const NodeContext& node, Arguments&... remaining) {
-    constexpr size_t args_len = sizeof...(Arguments);
-    TF_RETURN_IF_ERROR(ValidateInputCount(node, args_len));
-    return detail::GetInputNodes(node, 0, remaining...);
-}
-
-void TFTensorShapeToNGraphShape(const ::tensorflow::TensorShapeProto& tf_shape, ov::PartialShape* ng_shape);
+void tf_shape_to_ngraph_shape(const ::tensorflow::TensorShapeProto& tf_shape, ov::PartialShape* ng_shape);
 
 template <typename T>
-static void GetStaticInputVector(const ov::frontend::tf::NodeContext& node,
-                                 int64_t input_index,
-                                 std::vector<T>* vector) {
-    ov::Output<ov::Node> ng_input = node.get_ng_input(input_index);
-    auto constant = std::dynamic_pointer_cast<ov::opset8::Constant>(ng_input.get_node_shared_ptr());
-    FRONT_END_GENERAL_CHECK(constant != nullptr, "Node ", node.get_name(), " can't be casted to Constant.");
-    *vector = constant->cast_vector<T>();
+void get_static_input_vec(const NodeContext& node, int64_t input_index, std::vector<T>* vector) {
```

Comment:
propose to rename to `get_constant_input`

Reply:
I propose to fix it in the next PR

Reply:
fixed in https://github.com/openvinotoolkit/openvino/pull/8477